src/panhandler/views.py

```diff
@@ -1011,7 +1011,7 @@ def generate_dynamic_form(self, data=None) -> Form:
         # running = ('running', 'Running')
         # choices_list.append(candidate)
         # choices_list.append(running)
-        # dynamic_form.fields['config_source'] = forms.ChoiceField(widget=forms.Select, choices=tuple(choices_list),
+        # dynamic_form.fields['config_source'] = fields.ChoiceField(widget=forms.Select, choices=tuple(choices_list),
```

Comment:
might be needed, but you should add this as a different PR is necessary (doesn't look like it though)

Reply:
I'm not sure how that even got changed, I'll change it back